app/src/main/res/layout/activity_login.xml

```diff
@@ -124,4 +22,68 @@
         app:labelVisibilityMode="labeled"
         app:layout_constraintBottom_toBottomOf="parent"
         app:menu="@menu/bottom_navigation_menu" />
+
+    <EditText
+        android:id="@+id/email_field"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="87dp"
+        android:layout_marginTop="50dp"
+        android:layout_marginEnd="88dp"
+        android:layout_marginBottom="10dp"
+        android:ems="10"
+        android:hint="@string/email"
+        android:inputType="textEmailAddress"
+        app:layout_constraintBottom_toTopOf="@+id/password_field"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        android:autofillHints="Email" />
+
+    <EditText
+        android:id="@+id/password_field"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="87dp"
+        android:layout_marginTop="10dp"
+        android:layout_marginEnd="88dp"
+        android:layout_marginBottom="10dp"
+        android:ems="10"
+        android:hint="@string/password"
```

Comment:
I was thinking that Android probably has some default strings for "password", "login", "register" such kind of words

Reply:
From what I see, I didn't find them. Do you know where these default strings are stored ?

Reply:
https://stackoverflow.com/questions/12492084/how-do-i-use-default-android-resources-in-my-layout-xml

I didn't test in the code, but you can try this, otherwise just keep it as it is